src/es/validation.php

```diff
@@ -174,6 +174,7 @@
         'password_confirmation' => 'confirmación de la contraseña',
         'phone'                 => 'teléfono',
         'price'                 => 'precio',
+        'role'                  => 'rol',
```

Comment:
**ICYMI**: **role** was mentioned in json file.

Reply:
I added this attribute (`role`) to the `validation.php`  file so that I can correctly translate the error message when I don't select any role when adding a new member to the team. Otherwise you would get the **El campo role es obligatorio.** message which is incorrect in Spanish. The correct message is **El campo rol es obligatorio.**

![RoleValidation](https://user-images.githubusercontent.com/9275870/104242587-6209c100-542d-11eb-956d-90c89328151a.PNG)